pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -99,13 +103,16 @@ func NewManager(
 	reconcilePeriod time.Duration,
 	machineInfo *cadvisorapi.MachineInfo,
 	nodeAllocatableReservation v1.ResourceList,
+	stateFileDirecory string,
 ) (Manager, error) {
 	var policy Policy
+	var stateHandle state.State
 
 	switch policyName(cpuPolicyName) {
 
 	case PolicyNone:
 		policy = NewNonePolicy()
+		stateHandle = state.NewMemoryState()
```

Comment:
@flyingcougar if you want to fix this as part of this PR, that's OK too. @balajismaniam and I discussed it offline and think it would make sense to also use the file backed state for the none policy. That way, in step 4 above, we panic and tell the operator to delete the state file. WDYT?

Reply:
@ConnorDoyle @balajismaniam - what about panic here:

https://github.com/intelsdi-x/kubernetes/blob/826f3d93f70bd6ef01002d231482f6e20fc0f35b/pkg/kubelet/cm/cpumanager/state/state_file.go#L92-L94

Reply:
@ConnorDoyle @balajismaniam - what about panic here:
https://github.com/intelsdi-x/kubernetes/blob/826f3d93f70bd6ef01002d231482f6e20fc0f35b/pkg/kubelet/cm/cpumanager/state/state_file.go#L92-L94
Then we dont need file stat in none policy.

Reply:
The problem is that we need to use the file state in the none policy to execute that code! Otherwise we'll never read/write the policy name in the file. Please correct me if I'm wrong, were you able to reproduce the issue described above?

Reply:
Added back LGTM; let's take this up in a new PR.

Reply:
Tracking this in issue #55736